### PR TITLE
Task/cooks 328 new analytics report top 3 features per county

### PR DESCRIPTION
**Pull request by sophia-massie**
## Overview: ##
When a county is selected in Analytics, show the counties top 3 maltreatment risk factors with their associated rank (1-3), demographic feature name, and whether the feature has a negative or positive correlation to increased maltreatment

## Related Jira tickets: ##

* [COOKS-328](https://jira.tacc.utexas.edu/browse/COOKS-328)

## Summary of Changes: ##

Created new Analytics Predictive Table component. Added to the chart instructions component and made a new chart type to have the displayed explanation of the table provided by the data team. Updated the api.py file to catch an error if no data is available for that selected county.

## Testing Steps: ##
1. Go to https://cep.test/protx/dash/analytics and choose a county
2. Click around until you find a county with no data to ensure the proper display is shown. Make sure the data and table populates for counties that do have top three risk demographic data as well.

## UI Photos:
Display when there's no data for that county
Screen Shot 2022-10-20 at 12 54 30 PM

Display when there is data for that county

Screen Shot 2022-10-20 at 12 53 55 PM

## Notes: ##
For @nathanfranklin :

Should I change the protx-client/src/redux/reducers/protx.reducers.js file back to 
`export const initialAnalyticsState = {
  loading: true,`

Wasn't sure if this was a bug or if it was just for our testing yesterday. Thank you!

### CODE REVIEW

File: protx-client/src/components/ProTx/components/charts/AnalyticsPredictiveTable.jsx [hunk 1]
```diff
@@ -19,12 +21,18 @@ function AnalyticsPredictiveTable({geography, selectedGeographicFeature}) {
...
   const analytics = useSelector(
     state => state.protxAnalytics
   );
-  const showPlot = false; // Hide the plot while in dev.
+
+  const data = useSelector(
+    state => state.protx.data
+  );
+
+  const showPlot = true; // Hide the plot while in dev.
```
Comment (nathanfranklin) [inline protx-client/src/components/ProTx/components/charts/AnalyticsPredictiveTable.jsx:35]: can drop `showPlot` and `plotState`

Comment (sophia-massie) [reply protx-client/src/components/ProTx/components/charts/AnalyticsPredictiveTable.jsx]: Ah yes, meant to delete that.

**Review by nathanfranklin — APPROVED** [review]
LGTM. 👍 

---------

Only thing that could be altered is the width of the table so that it fills the space. But that can be addressed before merging or as a follow on PR>

**Review by ryamase — APPROVED** [review]
LGTM! 👍

**Pull request merged**